pkg/apis/ceph.rook.io/v1/validate_resources_test.go

```diff
@@ -0,0 +1,88 @@
+package v1
+
+import (
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestCephClusterValidateCreate(t *testing.T) {
+	c := &CephCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph",
+			Namespace: "rook-ceph",
+		},
+		Spec: ClusterSpec{
+			DataDirHostPath: "/var/lib/rook",
+		},
+	}
+	err := c.ValidateCreate()
+	assert.NoError(t, err)
+	c.Spec.External.Enable = true
+	c.Spec.Monitoring = MonitoringSpec{
+		Enabled:        false,
```

Comment:
false is the default, no need to set it

Reply:
Oops, it was supposed to be true. I've fixed it